README.md

```diff
@@ -2,28 +2,75 @@
 
 Merge Gatekeeper provides extra control for Pull Request management.
 
-## 🌄 What does Merge Gatekeeper provide, and Why?
+## 🌄 What Does Merge Gatekeeper Provide, and Why?
 
 <!-- == imptr: background / begin from: ./docs/details.md#[background] == -->
 
-Pull Request plays a significant part in day-to-day development, and making sure all the merges are controled is essential for building robust system. GitHub provides some control over CI, reviews, etc., but there are some limitations with handling special cases.
+Pull Request plays a significant part in day-to-day development, and it is essential to ensure all merges are well controlled and managed to build robust system. GitHub provides controls over CI, reviews, etc., but there are some limitations around handling specific use cases. Merge Gatekeeper helps overcome those by adding extra controls, such as monorepo friendly job check.
 
-At UPSIDER, we have a few internal repositories set up with a monorepo structure, where there are many types of code in the single repository. This comes with its own pros and cons, but with GitHub Action and merge control, there is no way to specify "Ensure Go build and test pass _if and only if_ Go code is updated", or "Ensure E2E tests are run and successful _if and only if_ frontend code is updated". Because of this limitation, we would either need to run all the CI jobs for all the code for any Pull Requests, or do not set any limitation based on the CI status. <sup>(\*1)</sup>
+At UPSIDER, we have a few internal repositories set up with a monorepo structure, with many types of code in a single repository. This comes with its own pros and cons, but one difficulty is how we end up with various CI jobs, which only run for changes that touch relevant files. With GitHub's branch protection, there is no way to specify "Ensure Go build and test pass _if and only if_ Go code is updated", or "Ensure E2E tests are run and successful _if and only if_ frontend code is updated". Because of this limitation, we would either need to run all the CI jobs for any Pull Requests, or do not set any limitation based on the CI status. <sup><sub><sup>(\*1)</sup></sub></sup>
```

Comment:
what about adding a sentence to describe how the job check of GitHub's branch protection can be set up?

I was unsure why we can't handle such cases until looking at the screenshot below. (the protection needs to specify job name and that's why such setup is not supported)

Reply:
I didn't want to go too much into the details, but it's worth adding such a note indeed. I'll work on that.

Reply:
I tried adding one sentence of "This is due to the GitHub branch protection design to specify a list of jobs to pass, which is only driven by the target branch name, regardless of change details." with 929be2f. Does this look better now?

Reply:
thanks, it looks better to me